src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs

```diff
@@ -298,7 +298,9 @@
         :toolbar (add-right
                   [comps/Button
                    {:text "Create new method..."
-                    :onClick #(modal/push-modal [create/CreateMethodDialog])}])
+                    :onClick #(modal/push-modal [create/CreateMethodDialog {:on-created (when (:in-workspace? props)
```

Comment:
Another option:
```cljs
[create/CreateMethodDialog
  {:on-created (fn [type id]
                 (if (:in-workspace? props)
                   ((:on-selected props) type id)
                   (nav/go-to-path type id)))}]
```
Then you wouldn't have to pass `:in-workspace?` down and you can just call `:on-created` in `CreateMethodDialog`. Also, line-length.